packages/flutter/lib/src/painting/continuous_rectangle_border.dart

```diff
@@ -26,27 +46,158 @@ import 'edge_insets.dart';
+    double rightX(double x) { return centerX - x * limitedRadius + width / 2; }
+    double topY(double y) { return centerY + y * limitedRadius - height / 2; }
+    double bottomY(double y) { return centerY - y * limitedRadius + height / 2; }
+
+    // Renders the default super elliptical rounded rect shape where there are
+    // 4 straight edges and 4 90º corners. Approximately renders a super ellipse
+    // with n value of 5.
+    Path roundedRect () {
+      return Path()
+        ..moveTo(leftX(1.52866483), topY(0))
+        ..lineTo(rightX(1.52866471), topY(0))
+        ..cubicTo(rightX(1.08849323), topY(0),
+            rightX(0.86840689), topY(0),
+            rightX(0.66993427), topY(0.06549600))
+        ..lineTo(rightX(0.63149399), topY(0.07491100))
+        ..cubicTo(rightX(0.37282392), topY(0.16905899),
+            rightX(0.16906013), topY(0.37282401),
+            rightX(0.07491176), topY(0.63149399))
+        ..cubicTo(rightX(0), topY(0.86840701),
+            rightX(0), topY(1.08849299),
+            rightX(0), topY(1.52866483))
+        ..lineTo(rightX(0), bottomY(1.52866471))
+        ..cubicTo(rightX(0), bottomY(1.08849323),
+            rightX(0), bottomY(0.86840689),
+            rightX(0.06549600), bottomY(0.66993427))
+        ..lineTo(rightX(0.07491100), bottomY(0.63149399))
+        ..cubicTo(rightX(0.16905899), bottomY(0.37282392),
+            rightX(0.37282401), bottomY(0.16906013),
+            rightX(0.63149399), bottomY(0.07491176))
+        ..cubicTo(rightX(0.86840701), bottomY(0),
+            rightX(1.08849299), bottomY(0),
+            rightX(1.52866483), bottomY(0))
+        ..lineTo(leftX(1.52866483), bottomY(0))
+        ..cubicTo(leftX(1.08849323), bottomY(0),
+            leftX(0.86840689), bottomY(0),
+            leftX(0.66993427), bottomY(0.06549600))
+        ..lineTo(leftX(0.63149399), bottomY(0.07491100))
+        ..cubicTo(leftX(0.37282392), bottomY(0.16905899),
+            leftX(0.16906013), bottomY(0.37282401),
+            leftX(0.07491176), bottomY(0.63149399))
+        ..cubicTo(leftX(0), bottomY(0.86840701),
+            leftX(0), bottomY(1.08849299),
+            leftX(0), bottomY(1.52866483))
+        ..lineTo(leftX(0), topY(1.52866471))
+        ..cubicTo(leftX(0), topY(1.08849323),
+            leftX(0), topY(0.86840689),
+            leftX(0.06549600), topY(0.66993427))
+        ..lineTo(leftX(0.07491100), topY(0.63149399))
+        ..cubicTo(leftX(0.16905899), topY(0.37282392),
+            leftX(0.37282401), topY(0.16906013),
+            leftX(0.63149399), topY(0.07491176))
+        ..cubicTo(leftX(0.86840701), topY(0),
+            leftX(1.08849299), topY(0),
+            leftX(1.52866483), topY(0))
+        ..close();
+    }
+
+    // The multiplier of the radius in comparison to the smallest edge length
+    // used to describe the minimum radius for this shape.
+    const double dynamicRadiusMinMultiplier = 2.0;
```

Comment:
Not sure what is the difference between this and maxMultiplier. From the class doc, it felt like 3 was the multiplier used.

Reply:
It's the multiplier in the case of an extreme aspect ratio and a small extent value.